pkg/multicluster/cluster_management.go

```diff
@@ -410,7 +410,7 @@ func JoinClusterByKubeConfig(ctx context.Context, cli client.Client, kubeconfigP
 		}
 	}
 	if cfg, ok := ctx.Value(KubeConfigContext).(*rest.Config); ok {
-		if err = SetClusterVersionInfo(ctx, cfg, clusterName); err != nil {
+		if err = SetClusterVersionInfo(ctx, cfg, clusterConfig.ClusterName); err != nil {
```

Comment:
clusterConfig.ClutserName usually is empty

Reply:
no, please see the picture below 
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/37976659/199926845-2094d09f-1e29-44f7-a81d-47b7fffc5d94.png">

Reply:
In theory, as long as the kubeconfig read is no problem, the cluster name in the cluster configuration cannot be empty.

Reply:
ping @Somefive , please review

Reply:
/cc @wonderflow